package.json

```diff
@@ -37,29 +37,27 @@
   },
   "homepage": "https://github.com/graphile/migrate#readme",
   "dependencies": {
-    "@types/chalk": "^2.2.0",
-    "@types/chokidar": "^2.1.3",
-    "@types/pg": "^7.11.1",
-    "chalk": "^2.4.2",
-    "chokidar": "^3.1.1",
-    "pg": ">=6.5 <8",
+    "@types/pg": "^7.14.1",
+    "chalk": "^3.0.0",
+    "chokidar": "^3.3.1",
+    "pg": "^7.18.1",
```

Comment:
Looks like `yarn upgrade --latest` converted `>=6.5 <8` to `^7.18.1`. Should I keep this change, or revert it?

Reply:
Please revert it 👍

Reply:
Done